python/lsst/pipe/tasks/calibrate.py

```diff
@@ -446,34 +460,41 @@ def runDataRef(self, dataRef, exposure=None, background=None, icSourceCat=None,
 
     def run(self, exposure, exposureIdInfo=None, background=None,
             icSourceCat=None):
-        """!Calibrate an exposure (science image or coadd)
+        """Calibrate an exposure (science image or coadd)
 
-        @param[in,out] exposure  exposure to calibrate (an
-            lsst.afw.image.ExposureF or similar);
+        Parameters
+        ----------
+        exposure : `lsst.afw.image.ExposureF`
+            exposure to calibrate (an lsst.afw.image.ExposureF or similar);
             in:
             - MaskedImage
             - Psf
+
             out:
             - MaskedImage has background subtracted
             - Wcs is replaced
             - Calib zero-point is set
-        @param[in] exposureIdInfo  ID info for exposure (an
-            lsst.obs.base.ExposureIdInfo) If not provided, returned
+
+        exposureIdInfo : `lsst.obs.base.ExposureIdInfo`
+            ID info for exposure. If not provided, returned
             SourceCatalog IDs will not be globally unique.
-        @param[in,out] background  background model already subtracted from
+        background : `lsst.afw.math.BackgroundList`
+            background model already subtracted from
             exposure (an lsst.afw.math.BackgroundList). May be None if no
             background has been subtracted, though that is unusual for
             calibration. A refined background model is output.
-        @param[in] icSourceCat  A SourceCatalog from CharacterizeImageTask
+        icSourceCat :
+            A SourceCatalog from CharacterizeImageTask
             from which we can copy some fields.
 
-        @return pipe_base Struct containing these fields:
-        - exposure  calibrate science exposure with refined WCS and Calib
-        - background  model of background subtracted from exposure (an
-          lsst.afw.math.BackgroundList)
-        - sourceCat  catalog of measured sources
-        - astromMatches  list of source/refObj matches from the astrometry
-          solver
+        Returns
+        -------
+        pipe_base Struct : `struct`
```

Comment:
When there's a nameless Struct returned, I think the preferred format is
```
       Returns
        -------
        result : `lsst.pipe.base.Struct`
           Result struct with components:

           - ``tempExprefList``: `list` of data references to tempExp.
           - ``weightList``: `list` of weightings.
           - ``imageScalerList``: `list` of image scalers.
```
Note in particular the spacing around the colons and how each component of the Struct has a type in backticks.